adapters/json_parse.go

```diff
@@ -125,3 +127,17 @@ func isArray(js *simplejson.Json, key string) bool {
 	}
 	return true
 }
+
+type jsonPath []string
+
+func (jp *jsonPath) UnmarshalJSON(b []byte) error {
+	strs := []string{}
+	var err error
+	if utils.IsQuoted(b) {
+		strs = strings.Split(string(utils.RemoveQuotes(b)), ".")
```

Comment:
Is it possible that you might ever want to match a string with a period within it?

Reply:
If you want that, you can always use the already existing array syntax; e.g. `{"path": ["a.", "b."]}`